handlers/handlers_test.go

```diff
@@ -448,6 +449,7 @@ func TestUnitHandlers(t *testing.T) {
 		})
 
 		Convey("filterable landing page returned if census config is false", func() {
+			const numOptsSummary = 50
```

Comment:
Why is it being set to 50 in this test? Shouldn't it be set to the same value as in live (i.e. 1000)?

Reply:
The test is mainly to prove that the filterable page will still load if `EnableCensusPages` is `false` and those pages (CMD) are currently limited to 50